exercises/kindergarten-garden/description.md

```diff
@@ -34,12 +34,12 @@ VR......................
 RG......................
 ```
 
-So in the row nearest the window, she has a violet and a radish; in the
-row behind that, she has a radish and some grass.
+In the row nearest the window, she has a violet and a radish.  In the
+second row she has a radish and some grass.
```

Comment:
"first" and "second" are used as a pair, but there is no "first".

"In the other row"?
or 
"In the first row nearest the window"

Reply:
Good eye.....